models/base/scratch/bigquery/snowplow_mobile_base_events_this_run.sql

```diff
@@ -0,0 +1,95 @@
+{{ 
+  config(
+    partition_by = {
+      "field": "collector_tstamp",
+      "data_type": "timestamp"
+    },
+    cluster_by=["event_name","screen_view_id"],
+    tags=["this_run"]
+  ) 
+}}
+
+{%- set lower_limit, upper_limit = snowplow_utils.return_limits_from_model(ref('snowplow_mobile_base_sessions_this_run'),
+                                                                          'start_tstamp',
+                                                                          'end_tstamp') %}
+
+{% set session_id = snowplow_mobile.get_session_id_path_sql(relation_alias='a') %}
+
+
+with events as (
+  select
+    -- screen view events
+    {{ snowplow_utils.get_optional_fields(
+          enabled=true,
+          col_prefix='unstruct_event_com_snowplowanalytics_mobile_screen_view_1_',
+          fields=screen_view_event_fields(),
+          relation=source('atomic','events'),
+          relation_alias='a') }},
+    -- screen context
+    {{ snowplow_utils.get_optional_fields(
+          enabled=var('snowplow__enable_screen_context', false),
+          col_prefix='contexts_com_snowplowanalytics_mobile_screen_1_',
+          fields=screen_context_fields(),
+          relation=source('atomic','events'),
+          relation_alias='a') }},
+    -- mobile context
+    {{ snowplow_utils.get_optional_fields(
+          enabled=var('snowplow__enable_mobile_context', false),
+          col_prefix='contexts_com_snowplowanalytics_snowplow_mobile_context_1_',
+          fields=mobile_context_fields(),
+          relation=source('atomic','events'),
+          relation_alias='a') }},
+    -- geo context
+    {{ snowplow_utils.get_optional_fields(
+          enabled=var('snowplow__enable_geolocation_context', false),
+          col_prefix='contexts_com_snowplowanalytics_snowplow_geolocation_context_1_',
+          fields=geo_context_fields(),
+          relation=source('atomic','events'),
+          relation_alias='a') }},
+    -- app context
+    {{ snowplow_utils.get_optional_fields(
+          enabled=var('snowplow__enable_application_context', false),
+          col_prefix='contexts_com_snowplowanalytics_mobile_application_1_',
+          fields=app_context_fields(),
+          relation=source('atomic','events'),
+          relation_alias='a') }},
+    -- session context
+    {{ snowplow_utils.get_optional_fields(
+          enabled=true,
+          col_prefix='contexts_com_snowplowanalytics_snowplow_client_session_1_',
+          fields=session_context_fields(),
+          relation=source('atomic','events'),
+          relation_alias='a') }},
+    a.*
+
+  from {{ source('atomic', 'events') }} as a
+  inner join {{ ref('snowplow_mobile_base_sessions_this_run') }} as b
+  on {{ session_id }} = b.session_id
+
+  where {{ snowplow_utils.timestamp_diff('a.collector_tstamp', 'b.start_tstamp', 'day')}} <= {{ var("snowplow__max_session_days", 3) }}
```

Comment:
Just noticed these two filters are incorrect too. I have described the issue here: https://github.com/snowplow/dbt-snowplow-web/issues/11

Very subtle difference but worth correcting I think. Seems like it applies to all 3 warehouses